lib/ClangImporter/ClangImporter.cpp

```diff
@@ -5552,7 +5552,7 @@ synthesizeDependentTypeThunkParamForwarding(AbstractFunctionDecl *afd, void *con
                                                       specParamTy);
     }
 
-    forwardingParams.push_back(Argument(SourceLoc(), Identifier(), argExpr));
+    forwardingParams.push_back(Argument::unlabeled(argExpr));
```

Comment:
There's a couple of these: as I went through and audited all `Argument` constructions to find the ones that needed internal labels, I changed any "nothing but an arg" constructions to use `::unlabeled` for _Maximum Semantics™_.